iris/views.py

```diff
@@ -118,6 +127,43 @@ def handle_feedback(message, l_session, course_id):
     emit('lecturer_recv', {'action': [action, s_feedback.count]}, room=course_id)
 
 
+@socketio.on('lecturer_keyword_new')
+def handle_new_keyword(message):
+    """
+
+    Receive json from lecturer,
+    add keywords and associated response to the database.
+    Push updates
+
+    """
+    course_id = message['course_id']
+    if course_id not in rooms():
+        return
+    keywords = message['keywords']
+    response = message['response']
+    print(message)
+    for keyword in keywords.split(','):
+        keyword = keyword.strip()
+        new_keyword = models.Response(keyword, course_id, response)
+        db.session.add(new_keyword)
```

Comment:
Should check if the keyword exists. Currently it crashes if it exists, maybe we should update the response text?